src/NuGet.Core/NuGet.Protocol/Model/V3SearchResults.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace NuGet.Protocol.Model
+{
+    public class V3SearchResults
```

Comment:
I think this class can be internal. It's only used temporarily in `RawSearchResults`, but it's never returned from a public method, or otherwise exposed outside the `NuGet.Protocol` assembly, right?

Reply:
You're right, changed to internal.